shared_model/backend/protobuf/common_objects/proto_common_objects_factory.hpp

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_PROTO_COMMON_OBJECTS_FACTORY_HPP
+#define IROHA_PROTO_COMMON_OBJECTS_FACTORY_HPP
+
+#include <regex>
+
+#include "backend/protobuf/common_objects/account.hpp"
+#include "backend/protobuf/common_objects/account_asset.hpp"
+#include "backend/protobuf/common_objects/asset.hpp"
+#include "backend/protobuf/common_objects/domain.hpp"
+#include "backend/protobuf/common_objects/peer.hpp"
+#include "backend/protobuf/common_objects/signature.hpp"
+#include "common/result.hpp"
+#include "interfaces/common_objects/common_objects_factory.hpp"
```

Comment:
Some of includes in this file are a subset of includes in ```common_objects_factory.hpp```. Maybe remove them? Such includes are ```account```, ```domain```, ```peer``` and ```result```